containers/pax-exam-container-forked/src/main/java/org/ops4j/pax/exam/forked/ForkedFrameworkFactory.java

```diff
@@ -105,9 +109,7 @@ public void setFrameworkFactory(FrameworkFactory frameworkFactory) {
     public RemoteFramework fork(List<String> vmArgs, Map<String, String> systemProperties,
         Map<String, Object> frameworkProperties, List<String> beforeFrameworkClasspath,
         List<String> afterFrameworkClasspath) {
-        // TODO make port range configurable
-        FreePort freePort = new FreePort(21000, 21099);
-        port = freePort.getPort();
+        port = Integer.parseInt(System.getProperty(EXAM_FORKED_INVOKER_PORT, String.valueOf(getFreePort().getPort())));
```

Comment:
I think we should really check here if the system property is set and then use this. Otherwise, always ports are scanned and the the result is thrown away!

Reply:
agree, done